pyradiosky/skymodel.py

```diff
@@ -1104,10 +1324,10 @@ def source_cuts(
             done using lst, and the lsts are calculated with astropy, the
             required buffer should _not_ drift with time since the J2000 epoch.
             The default buffer has been tested around julian date 2457458.0.
-        min_flux : float
-            Minimum stokes I flux to select [Jy]
+        min_flux : Quantity
```

Comment:
Yes, it should list Quantity as an option. I actually had a question about this, I'm not sure if we want to actually deprecate passing a float (which we say is assumed to be in Jy) because I wasn't sure how the requirement to make it a Quantity would affect the pyuvsim yaml specification.

Reply:
In pyuvsim, the source_cuts function is run with floats for min/max_flux in Jy. I don't see any reason to deprecate passing in a float, as long as it's clear what that means.